docs/gtm-interchain.md

```diff
@@ -0,0 +1,73 @@
+
+The IBC and Gaia go-to-market initiatives will be about direct engagement with the products that directly validate the utility of the Hub and IBC. This initiative encompasses the utility of various interoperable and cross-chain Cosmos entities such as application-specific blockchains, Bridges, Relayers, Validators, etc. More importantly, we envision that the market will quickly grasp the meaning and relevance of cross-chain interoperability of which Cosmos is a leading contender.
+
+**How we work**
+
+These discussions will take place during weekly Gaia team calls on Fridays. The Hub will determine the value proposition of IBC and how it is integrated into Gaia and how it will serve the larger ecosystem of interconnected blockchains. These discussions will continue to take place with key stakeholders during the bi-weekly Cosmos multi entity call. Importantly, the purpose of the bi-weekly multi entity call will be to consider all technical aspects of IBC launch from a strategic perspective (outlined in &#39;What&#39;s Needed to Successfully Go to Market?&#39;). This will inform the agenda.
+
+Leading up to Stargate, we can consider that Gaia Development was led by Chris Goes (Interchain Berlin) and go-to-market was led by Zaki (Iqlusion). Moving forward, we are aiming to organize Gaia Development led by Shahan Khatchadourian (Interchain Berlin) and go-to-market led by Jelena Djuric (Informal Systems). Both Shahan and Jelena are being onboarded into these roles with the support of the previous leads.
+
+Execution and completion will be tracked via a meta-project dashboard (Gaia Roadmap) that incorporates the high level objectives for each phase roll out. Each phase (i.e. Phase 1 - Accounts UX, Phase 2 - Interchain Interchange, etc) will have their own project dashboard that will link to the repos, issues, PRs, etc, where development is taking place.
+
+&quot;_Never doubt that a small group of thoughtful, committed, citizens can change the world. Indeed, it is the only thing that ever has.&quot; - Margaret Mead_
+
+**What&#39;s Needed to Successfully Go to Market?**
+
+1. **Project management**
+
+In sum, project management will focus on surfacing issues, dependencies, relative roadmaps, and coordinating across all stakeholders. Questions that will be considered throughout each phase of the Gaia roadmap:
+
+- What is missing from an integration points perspective with end users?
+
+- Which relationships are missing?
+- What&#39;s the highest priority? Least priority?
+
+2. **Market validation**
```

Comment:
Are we seeking partners as a component of validation?

Reply:
yes, i implied this i think but added line on this to be explicit